core/controllers/base.py

```diff
@@ -277,7 +297,14 @@ def render_json(self, values):
         self.response.write('%s%s' % (feconf.XSSI_PREFIX, json_output))
 
     def render_downloadable_file(self, values, filename, content_type):
-        """Prepares downloadable content to be sent to the client."""
+        """Prepares downloadable content to be sent to the client.
+
+        Args:
+            values: dict. The key-value pairs to encode 
+                in the downloadable file response.
+            filename: str. Name of file to be given.
+            content_type: str. Type of content to be passed.
```

Comment:
The content type of the downloaded file.

(Note: try to be specific. "to be passed" is vague -- to be passed to who?)

Reply:
okay sir sorry for that, i'll keep this in mind.